src/logic/proposals/hooks/history-proposals.ts

```diff
@@ -62,17 +62,18 @@ const useLoadHistoryProposals = () => {
       };
     });
     if (!isGoSuccess(goResponse)) {
-      notifications.error({ message: messages.FAILED_TO_LOAD_PROPOSALS, errorOrMessage: goResponse[GO_ERROR_INDEX] });
-      return;
+      return notifications.error({
+        message: messages.FAILED_TO_LOAD_PROPOSALS,
+        errorOrMessage: goResponse[GO_ERROR_INDEX],
+      });
     }
     const proposals = goResponse[GO_RESULT_INDEX];
 
     setChainData(
       'Load history proposals',
       updateImmutablyCurried((state) => {
         if (!state.proposals) {
-          state.proposals = proposals;
-          return;
+          return (state.proposals = proposals);
```

Comment:
I'm fine with returning the notification ID but having assignment and returning on the same line is a bit weird imo. I don't know if I've ever seen something like this before

Reply:
:) Remind me of my competitive programming career. It's actually an antipattern in practice, but I don't mind this much.

But since you consider it wierd, I'll revert to `return` on single line.